static/app/components/onboarding/gettingStartedDoc/onboardingLayout.tsx

```diff
@@ -131,9 +138,36 @@ export function OnboardingLayout({
         </Header>
         <Divider withBottomMargin />
         <Steps>
-          {steps.map(step => (
-            <Step key={step.title ?? step.type} {...step} />
-          ))}
+          {steps.map(step =>
+            step.isOptional && step.isReplayConfigStep ? (
```

Comment:
if the step is optional, we have to shove the replay config toggle inside the step's `additionalInfo` to hide them